web/modeshape-web-explorer/pom.xml

```diff
@@ -57,7 +57,12 @@
             <artifactId>gwt-user</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>com.googlecode.gwtmosaic</groupId>
+            <artifactId>gwt-incubator</artifactId>
+            <version>0.4.0-rc2</version>
+            <type>jar</type>
+        </dependency>
```

Comment:
Makes sense but unfortunately I did not find anything final available. This dependency is used only for single class `Canvas` which allows to draw on the page.